lib/src/windows_single_instance.dart

```diff
@@ -121,15 +123,15 @@ class WindowsSingleInstance {
     }
 
     // No callback so don't bother starting pipe
-    if (onSecondWindow == null) {
+    if (onSecondWindow == null && bringWindowToFront == false) {
       return;
     }
 
     final reader = ReceivePort()
       ..listen((dynamic msg) {
         if (msg is List) {
-          onSecondWindow(msg.map((o) => o.toString()).toList());
-          _bringWindowToFront();
+          onSecondWindow!(msg.map((o) => o.toString()).toList());
```

Comment:
This needs to use a ? instead of a !, else there will be an error if onSecondWindow is null